### DISCUSSION TITLE
Filter for failure doesn't include startup failure

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#58143] [Actions] [answered]
### Select Topic Area

Question

### Body

The https://github.com/community/community/actions/workflows/issue_projects_labeler.yml workflow is horribly broken at the moment (https://github.com/community/community/commit/861a04a2da53a507200f1bd2f1ff2ca2be6d9e14#commitcomment-118186705). But ignoring that, let's talk about filtering Workflows...

At the moment there are 60 runs in https://github.com/community/community/actions/workflows/issue_projects_labeler.yml

<img width="1705" alt="image" src="https://github.com/community/community/assets/2119212/8fbacefa-5750-4353-a983-2ef534022cd9">

A naive user might expect that if they select `failure` they'd get those 60 failures...
<img width="335" alt="image" src="https://github.com/community/community/assets/2119212/91d9cbbd-2ea8-4f55-8be1-4ea70be44c4c">

https://github.com/community/community/actions/workflows/issue_projects_labeler.yml?query=is%3Afailure
<img width="1703" alt="image" src="https://github.com/community/community/assets/2119212/24ba367c-93ad-470c-8b4f-5cde6f3c0d51">

Nope, just one: https://github.com/community/community/actions/runs/5271963656
> Annotations
1 error
[label-templated-issue](https://github.com/community/community/actions/runs/5271963656/jobs/9533578547#step:2:32)
Unhandled error: TypeError: Cannot read properties of undefined (reading 'body')

But it isn't that you can't filter for these things, they appear if you use `completed`:

https://github.com/community/community/actions/workflows/issue_projects_labeler.yml?query=is%3Acompleted
<img width="1703" alt="image" src="https://github.com/community/community/assets/2119212/b3fe1b8b-025f-4638-b4f3-6962b1fa2114">

The problem is that these errors are `startup failure`

https://github.com/community/community/actions/runs/5279007208
> Annotations
1 error
Invalid workflow file: .github/workflows/issue_projects_labeler.yml#L8
The workflow is not valid. .github/workflows/issue_projects_labeler.yml (Line: 8, Col: 3): Unexpected value 'discussion

And that should probably be a separate filter item, but isn't available.

---

Please either expose a filter for `Startup Failure` or merge it into the filter for `Failure`

**Comment by jsoref** [comment]
Here's failure #61 https://github.com/community/community/actions/runs/5279176520

**Comment by sebastian-fredriksson-bernholtz** [comment]
This is extremely frustrating. You can't add any kind of general notifications to workflows that fail to start because nothing will run. And if they fail, you can't filter for them? 

This makes it super difficult to monitor workflow to make sure that they are all running correctly...

Please, give us a way to check for failed workflow runs!!!

**Comment by jsoref** [comment, marked as answer]
- Refiled as https://github.com/community/community/discussions/181848
